ArcGISRuntimeSDKQt_CppSamples/UtilityNetwork/CreateLoadReport/README.metadata.json

```diff
@@ -0,0 +1,58 @@
+{
+    "category": "UtilityNetwork",
+    "description": "Create a simple electric distribution report that displays the count of customers and total load per phase by tracing downstream from a given point.",
+    "ignore": false,
+    "images": [
+        "screenshot.png"
+    ],
+    "keywords": [
+        "condition barriers",
+        "downstream trace",
+        "network analysis",
+        "subnetwork trace",
+        "trace configuration",
+        "traversability",
+        "upstream trace",
+        "utility network",
+        "validate consistency",
```

Comment:
not sure if the website build is resilient to a trailing , like this. Probably best to remove unless you know otherwise